huaweicloud/services/acceptance/gaussdb/resource_huaweicloud_gaussdb_mysql_instance_test.go

```diff
@@ -272,6 +276,8 @@ resource "huaweicloud_gaussdb_mysql_instance" "test" {
   private_write_ip         = "192.168.0.156"
   port                     = "8888"
   private_dns_name_prefix  = "testprivatednsname"
+  maintain_begin           = "08:00"
+  maintain_end             = "11:00"
```

Comment:
in the doc, there is note said the interval between them must be four hours.
there are only 3 hours between 08:00 and 11:00

Reply:
fixed the doc